### PR TITLE
refactor: remove duplicated fetcher fn

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/wallet/actions/runs/5110129218).

Refactored to allow reuse of the fetcher

### CODE REVIEW

**Review by fbwoolf — APPROVED** [review]

File: src/shared/utils.ts [hunk 1]
```diff
@@ -33,3 +33,7 @@ export function isTypedArray(val: unknown): val is Uint8Array {
 
 // eslint-disable-next-line @typescript-eslint/no-empty-function
 export function noop() {}
+
+export function arrayIfNotArray<T>(value: T | T[]): T[] {
```
Comment (alter-eggo) [inline src/shared/utils.ts]: wdyt?
Screenshot 2023-05-29 at 12 33 03

**Review by alter-eggo — APPROVED** [review]

File: src/shared/utils.ts (hunk 1 above)
Comment (kyranjamie) [reply src/shared/utils.ts]: Yes, I like this name better 👍🏼

Comment (kyranjamie) [reply src/shared/utils.ts]: What was the prompt?

Comment (alter-eggo) [reply src/shared/utils.ts]:
```
how to better name this function export function arrayIfNotArray<T>(value: T | T[]): T[] {
  return Array.isArray(value) ? value : [value];
}
```

**Pull request merged**